src/display/annotation_layer.js

```diff
@@ -239,6 +239,38 @@ class AnnotationElement {
     return container;
   }
 
+  /**
+   * Create areas for the quadPoints
+   *
+   * @private
+   * @memberof AnnotationElement
+   */
+  _createAreas() {
+    if (this.data.quadPoints) {
```

Comment:
You can reduce the overall indentation by using an early return instead here, i.e.
```javascript
if (!this.data.quadPoints) {
  return;
}
// The rest of the method goes here...
```